psetv2/input.go

```diff
@@ -205,9 +205,6 @@ func (i *Input) HasIssuanceBlinded() bool {
 }
 
 func (i *Input) HasReissuance() bool {
-	if len(i.IssuanceBlindingNonce) > 0 {
-		return false
-	}
 	return !bytes.Equal(i.IssuanceBlindingNonce, transaction.Zero[:])
```

Comment:
let's keep the original check and fix it. the condition should be `<=` not `>`. If no blinding nonce is set the inputs hasn't got any reissuance.